steps/previsit/first-attempt.tsx

```diff
@@ -0,0 +1,129 @@
+import formatDate from "date-fns/format";
+import { Heading } from "lbh-frontend-react";
+import {
+  FieldsetLegend,
+  HeadingLevels,
+  Paragraph
+} from "lbh-frontend-react/components";
+import React from "react";
+import {
+  ComponentDatabaseMap,
+  ComponentWrapper,
+  DynamicComponent,
+  StaticComponent
+} from "remultiform/component-wrapper";
+import { Checkboxes, CheckboxesProps } from "../../components/Checkboxes";
+import { makeSubmit } from "../../components/makeSubmit";
+import { TextArea } from "../../components/TextArea";
+import ProcessDatabaseSchema from "../../storage/ProcessDatabaseSchema";
+import processRef from "../../storage/processRef";
+import PageSlugs from "../PageSlugs";
+import PageTitles from "../PageTitles";
+
+const step = {
+  title: PageTitles.FirstAttempt,
+  heading: "Unable to enter the property",
+  step: {
+    slug: PageSlugs.FirstAttempt,
+    nextSlug: PageSlugs.Loading,
```

Comment:
Going back to the loading page will cause the data to be blown away. This might be ok for now, but I think we probably need an equivalent of the `pause` and `paused` pages for this part of the journey.

Reply:
I think for now, let's actually send users to the `PageSlugs.Pause`.